tyndale-bulletin-fullnote-bibliography.csl

```diff
@@ -0,0 +1,1054 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="chicago" default-locale="en-GB">
+  <info>
+    <title>Tyndale Bulletin (full note, bibliography)</title>
```

Comment:
Sorry, wasn't clear here: if there's just one style, this should just be `Tyndale Bulletin`